signac/contrib/project.py

```diff
@@ -26,7 +26,7 @@
 from ..common.config import Config, get_config, load_config
 from ..core import json
 from ..core.h5store import H5StoreManager
-from ..core.jsondict import JSONDict
+from ..core.synced_collections.collection_json import BufferedJSONDict
```

Comment:
I think I'd like to lay the groundwork for re-structuring signac 2.0 and/or separating out the `synced_collections` package by moving `synced_collections` to the top level outside of `core`.